adios_support branch has build errors in adios2pio.cxx

On anlworkstation, building adios_support branch failed with error like:
```
tools/adios2pio/adios2pio.cxx: In function ‘void ConvertBPFile(std::__cxx11::string, std::__cxx11::string, int)’:
tools/adios2pio/adios2pio.cxx:162:9: error: ‘map’ is not a member of ‘std’
         std::map<std::string,int> decomp_map;
         ^~~
...
```

--- a/tools/adios2pio/adios2pio.cxx
+++ b/tools/adios2pio/adios2pio.cxx
@@ -3,6 +3,7 @@
 #include <iostream>
 #include <string>
 #include <vector>
+#include <map>
 #include <mpi.h>
 #include "pio.h"
 #include "adios_read.h"

